bin/knc.c

```diff
@@ -56,18 +56,25 @@ prefs_t prefs;
 /* BEGIN_DECLS */
 
 void	sig_handler(int);
+void	sig_set(int, void (*)(int), int);
```

Comment:
Well, OK, but it's a single source code file program, so they should all be static.  But whatever.

Reply:
As discussed, I am not going to make this change at this time, but we might choose to do so in a later PR.